logtail/logtail.go

```diff
@@ -437,14 +465,24 @@ func (l *Logger) send(jsonBlob []byte) (int, error) {
 
 // TODO: instead of allocating, this should probably just append
 // directly into the output log buffer.
-func (l *Logger) encodeText(buf []byte, skipClientTime bool, level int) []byte {
+func (l *Logger) encodeText(buf []byte, skipClientTime bool, instanceKey *string, instanceSequence int64, level int) []byte {
 	now := l.timeNow()
 
 	// Factor in JSON encoding overhead to try to only do one alloc
 	// in the make below (so appends don't resize the buffer).
 	overhead := 13
```

Comment:
Instead of magical length constants, use a `len`:
```go
overhead := len(`"logtail": `)
```
and so forth.

In fact, I'm not sure where 13 came from, so my suggestion might not be using the right string literal.

Reply:
I like this, switched to it (didn't realize the compiler would be smart enough to generate a constant, but it does -- https://go.godbolt.org/z/hPGoTbhrW).

I believe the 13 is due to `{"text": ""}\n`